### ISSUE TITLE
BOINC Manager on MacOS: missing keyboard shortcut for Preferences

### ISSUE THREAD

**Issue by EelcoV** [BOINC/boinc#5273]
Every MacOS application (with a Preferences menu item) accepts the keyboard shortcut Cmd-comma (⌘-,) for Preferences.
The BOINC Manager ignores the Cmd-comma shortcut.
It should open the Computing preferences window.

MacOS Ventura 13.4
BOINC Manager 7.20.4

**Comment by AenBleidd** [comment]
@CharlieFenton, is it something that should be fixed for consistency with other MacOS applications for better user experience?

**Comment by Signalshifter** [comment]
I dealt with this issue when I first loaded Boinic Manager.  I ran Boinc Manager on my PC's for more than 20 years. when I transitioned to macOS and a Mac Studio I loaded up Bionic Manager right away. I could not configure it like I did the Windows version. The buttons to do this were not located at the top of the simple Boinc screen.  It was then pointed out to me these controls are found by pointing the curser at the Boinic Manager name in the upper left hand corner of the screen and right clicking your mouse, next right click on preferences, and this will open the controls.  I tried the Command - Comma command and as you pointed out it does not work.  But given the way to get too preferences is setup I am good to go.
